src/components/renders/bootstrap/select/select-render.tsx

```diff
@@ -78,13 +79,16 @@ export class SelectRender implements Renderer {
           select.value = this.component.value;
         }
       };
+      if (this.component.suggest) {
+        this.datalistId = `${this.selectId}__datalist`;
```

Comment:
Just an observation: instead of a property, this could have been a local variable inside the `render` method as it's not used outside of the method.